src/editor.js

```diff
@@ -841,12 +841,13 @@ function resetToolBoxSizing(resizeDelay, centerBlocks) {
         // Update the Blockly editor canvas to use the new space
         if (Blockly.mainWorkspace && blocklyDiv[0].style.display !== 'none') {
             Blockly.svgResize(Blockly.mainWorkspace);
```

Comment:
Move this inside a test to make sure the Blockly Workspace exists first - had a race condition pop up during testing on the newProject initialization, this solved it.